site/src/content/docs/ref/packages.mdx

```diff
@@ -20,17 +20,19 @@ Throughout the rest of the documentation, we will refer to the `ZarfInitConfig`
 
 ### `ZarfInitConfig`
 
-The init package is used to initialize a cluster, making it ready for deployment of other Zarf packages. It must be executed once on each cluster that you want to deploy another package onto, even if multiple clusters share the same host. For additional information on the init package, we provide detailed documentation on the Zarf ['init' package page](/ref/init-package/).
+The init package is used to initialize a cluster, making it ready for deployment of other Zarf packages. It must be executed once on each cluster that you want to deploy another package onto, even if multiple clusters share the same host.
 
-If there is no running cluster, the init package can be used to create one. It has a deployable K3s cluster component that can be optionally deployed on your machine. Usually, an init package is the first Zarf Package to be deployed on a cluster as other packages often depend on the services installed or configured by the init package. If you want to install a K8s cluster with Zarf, but you don't want to use K3s as your cluster, you will need to create or find another Zarf Package that will stand up your cluster before you run the zarf init command.
+If there is no running cluster, the init package can be used to create one. It has a deployable K3s cluster component that can be optionally deployed on your machine. To install a K8s distribution other than k3s, finding or creating a custom Zarf package is necessary.
```

Comment:
nit: our style guide says that we prefer to use `K3s` with a capital K and `Zarf Package` with a capital P

https://github.com/defenseunicorns/zarf/blob/main/site/src/content/docs/contribute/style-guide.mdx

```suggestion
If there is no running cluster, the init package can be used to create one. It has a deployable K3s cluster component that can be optionally deployed on your machine. To install a K8s distribution other than K3s, finding or creating a custom Zarf Package is necessary.
```